gpflow/base.py

```diff
@@ -303,7 +303,7 @@ def _cast_to_dtype(value: VariableData, dtype: Optional[DType] = None) -> tf.Ten
     if dtype is None:
         dtype = default_float()
     if tf.is_tensor(value):
-        # TODO(awav) TF2.2 resolves issue with cast.
+        # NOTE(awav) TF2.2 resolves issue with cast.
```

Comment:
Is this not something that we want to do ? why not leave it as a todo?

Reply:
`TODO:` - intention to do something, because we have control over it. `NOTE:` - we have to live with it, even if this is for a short time.